search/lib/cmr.js

```diff
@@ -49,14 +50,28 @@ async function findCollections (params = {}) {
   return response.data.feed.entry;
 }
 
+async function stacIdToCmrCollectionId (providerId, stacId) {
+  const cacheKey = `stacId_${stacId}`;
```

Comment:
Why do we prefix the `stacId `with `stacId_` as the cache key? I don't see any value in doing that.

Reply:
I guess it's not really necessary.  When I first implemented the cache in the `cmrCollectionIdToStacId` I added that as a way to namespace the cache in case the cache was used elsewhere.
Now it's used to map from CMR collection ID -> STAC ID (`cmrCollectionIdToStacId`) and STAC ID -> CMR Collection ID (`stacIdToCmrCollectionId`), but you're right the additional namespace doesn't do much because there's no conflict between those var names